bin/rrdd/version.ml

```diff
@@ -0,0 +1 @@
+let version = "0.12.0"
```

Comment:
Instead of hardcoding the version might be better to use Dune to set it.
For XAPI this didn't work because we wanted the version in a particular format, but for other daemons/executables it should work sufficiently well: https://dune.readthedocs.io/en/stable/dune-libs.html#build-info
For RPM builds we can append a line to the `dune-project` file containing the version (so that this works when you build from git). And when building from opam (and with dune prior to install) you can just handle the `None` case and return `n/a` as in the example there.

Reply:
I think we want some uniformity here across daemons. So maybe leave this for later rather than trying to improve this during an already busy PR?